src/test/java/seedu/cakecollate/commons/util/StringUtilTest.java

```diff
@@ -174,4 +176,31 @@ public void getDetails_nullGiven_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> StringUtil.getDetails(null));
     }
 
+    // --------------- Tests for capitalise -----------------------------------
+    @Test
+    public void capitaliseEachWordTest() {
+
+        assertEquals("Hello World", capitaliseEachWord("HELLO WORLD")); // all caps
+        assertEquals("Hello World", capitaliseEachWord("hellO wOrLd")); // random caps
+        assertEquals("Hello World", capitaliseEachWord("hello world")); // all lower case
+        assertEquals("Hello World", capitaliseEachWord("Hello World")); // input same as output
+
+        // special characters preserved
+        // include standalone special characters, special characters sandwiched/at the start/at the end of word
+        assertEquals(capitaliseEachWord("Hello w#$% world ..@@& !hello"), "Hello W#$% World ..@@& !hello");
+
+        // numbers preserved (include numbers at start of word, numbers in middle of word, numbers beside letters)
+        assertEquals("Hello 78 World", capitaliseEachWord("hello 78 world"));
+        assertEquals("Hello 78world", capitaliseEachWord("hello 78world"));
+        assertEquals("Hello Wor34ld", capitaliseEachWord("hello wor34ld"));
+        assertEquals("Hello World12", capitaliseEachWord("hello world12"));
```

Comment:
Actually I removed numbers from the order item description so I think that we probably don't need these tests. But no harm in letting them be, probably good to have them in case capitaliseEachWord needs to be used for other features too.